### DISCUSSION TITLE
Check-only moves?

### DISCUSSION THREAD

**Discussion by HGMuller** [fairy-stockfish/Fairy-Stockfish#841] [Variants]
Is it possible to define moves that cannot capture, but can deliver check? In extended Betza notation I introduced the **k** for this, as an alternative to **m** and **c**. This made it for instance possible to describe the King-facing rule in Xiangqi with an extra **fkR** move on the Kings.

**Comment by ianfab** [comment]
This isn't supported yet, but would be a nice generalization of the king-facing rule, so I converted this discussion to issue #842 as a feature request.
